kpops/components/base_components/pipeline_component.py

```diff
@@ -75,6 +75,24 @@ def __init__(self, **kwargs) -> None:
         self.set_input_topics()
         self.set_output_topics()
 
+    def get_input_topics(self) -> list[str] | None:
```

Comment:
not really, I think makes more sense None, because things like a producer will not implement this and per default it will return None. This would make sense if we make the implementation mandatory